### PR TITLE
Bump path-to-regexp dependency to v0.1.10

### PR DESCRIPTION
**Pull request by watson**

### CODE REVIEW

**Comment by watson** [comment]
* **#4664** <a href="https://app.graphite.dev/github/pr/DataDog/dd-trace-js/4664?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @watson and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: yarn.lock [hunk 1]
```diff
@@ -3654,11 +3654,16 @@ path-parse@^1.0.7:
   resolved "https://registry.npmjs.org/path-parse/-/path-parse-1.0.7.tgz"
   integrity sha512-LDJzPVEEEPR+y48z93A0Ed0yXb8pAByGWo/k5YYdYgpY2/2EsOsksJrq7lOHxryrVOn1ejG6oAp8ahvOIQD8sw==
 
-path-to-regexp@0.1.7, path-to-regexp@^0.1.2:
+path-to-regexp@0.1.7:
   version "0.1.7"
```
Comment (simon-id) [inline yarn.lock:3658]: well 0.1.7 is still vulnerable according to the advisory no ?

File: yarn.lock [hunk 2]
```diff
@@ -3654,11 +3654,16 @@ path-parse@^1.0.7:
...
   resolved "https://registry.npmjs.org/path-to-regexp/-/path-to-regexp-0.1.7.tgz"
   integrity sha512-5DFkuoqlv1uYQKxy8omFBeJPQcdoE07Kv2sferDCrAq1ohOU+MSDswDIbnx3YAM60qIOnYa53wBhXW0EbMonrQ==
 
+path-to-regexp@^0.1.10:
+  version "0.1.10"
+  resolved "https://registry.yarnpkg.com/path-to-regexp/-/path-to-regexp-0.1.10.tgz#67e9108c5c0551b9e5326064387de4763c4d5f8b"
+  integrity sha512-7lf7qcQidTku0Gu3YDPc8DJ1q7OOucfa/BSsIwjuh56VU7katFvuM8hULfkwB3Fns/rsVF7PwPKVw1sl5KQS9w==
+
 path-to-regexp@^1.7.0:
   version "1.8.0"
```
Comment (simon-id) [inline yarn.lock:3668]: same thing for 1.8.0

File: yarn.lock (hunk 1 above)
Comment (watson) [reply yarn.lock:3658]: In theory yes, but this is part of a dev-dependency, so not critical and not blocking CI

File: yarn.lock (hunk 2 above)
Comment (watson) [reply yarn.lock:3668]: In theory yes, but this is part of a dev-dependency, so not critical and not blocking CI

Comment (simon-id) [reply yarn.lock:3668]: of nise which is dep of sinon yep

File: yarn.lock (hunk 1 above)
Comment (simon-id) [reply yarn.lock:3658]: Isn't this a dep of express ?

Comment (simon-id) [reply yarn.lock:3658]: which is a dev dep obviously nvm :facepalm:

**Review by simon-id — APPROVED** [review]

**Comment by thiagoribeir015** [comment]
hey @watson @simon-id , is there an ETA for rolling out a new version? 🙇🏻

**Comment by JasonKleban** [comment]
Note: path-to-regexp is up to version 8 now  https://github.com/pillarjs/path-to-regexp/releases

**Comment by watson** [comment]
@thiagoribeir015 we just published version 4.47.0 and 5.23.0 to npm an hour ago.

**Comment by thiagoribeir015** [comment]
Thanks a lot! 🙇🏻

**Pull request merged**